lib/index.js

```diff
@@ -193,7 +193,23 @@ Nightwatch.cli = function(callback) {
       throw new Error('Supplied callback argument needs to be a function.');
     }
 
-    callback(argv);
+    if (Array.isArray(argv.env)){
+      // Enable running tests in sequential mode for supported and provided environments
+      if (argv.parallel){
```

Comment:
I think we should be able to make it work in parallel mode as well.

Reply:
By enabling the parallel the mode, it starts behaving very strange.